pkg/scheduler/framework/plugins/interpodaffinity/scoring_test.go

```diff
@@ -22,7 +22,7 @@ import (
 	"strings"
 	"testing"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
revert the file.

Reply:
@Huang-Wei @ahg-g this change, it's taking on it's own when I try to save the file. Did you mean revert the file to a previous commit?

Reply:
@ahg-g just meant to discard this change.

Reply:
Done with the latest push